docs/remote_development.md

```diff
@@ -83,7 +83,14 @@ This file can be committed so others can use your deployed contracts.
 
 The [aggregator](../aggregator/) is a service that accepts transaction bundles (including those that contain a single transaction) and submits aggregations of these bundles to L2.
 
-Update the following values in `./aggregator/.env`
+```
+cd ./aggregator
+cp .env.example .env
+```
+> **Note: This .env file now uses `arbitrum` RPCs.**
```

Comment:
Could you remove this comment and the whitespace below it? Don't think it's needed as arbitrum is mentioned above and you can see the arbitrum goerli rpcs in the .env file